lib/sidekiq/honeycomb_middleware.rb

```diff
@@ -0,0 +1,24 @@
+module Sidekiq
+  class HoneycombMiddleware
+    # @param [Object] worker the worker instance
+    # @param [Hash] job the full job payload
+    #   * @see https://github.com/mperham/sidekiq/wiki/Job-Format
+    # @param [String] queue the name of the queue the job was pulled from
+    def call(worker, job, queue)
+      Honeycomb.start_span(name: "sidekiq.job") do |span|
```

Comment:
remember that your span names should be descriptive enough for you to figure out what they are; the service name of "sidekiq" would likely suffice and then you can use `name`s that vary based on `class` or similar.